src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/config.jelly

```diff
@@ -64,6 +64,10 @@
       <f:textbox clazz="required positive-number" default="1" />
     </f:entry>
 
+    <f:entry title="${%Scale Executors by Weight}" field="scaleExecutorsByWeight">
+      <f:checkbox />
+    </f:entry>
```

Comment:
Please add help page about this field, example is https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/src/main/resources/com/amazon/jenkins/ec2fleet/EC2FleetCloud/help-addNodeOnlyIfRunning.html